### PR TITLE
[DOC] Test & Score: Fixed and extended description for target classes

### PR DESCRIPTION
**Pull request by PrimozGodec**
#### Issue

The documentation was not clear about scores computing in case of more classes. 
There were also wrong info in the documentation.
https://github.com/biolab/orange3/issues/3098

##### Description of changes
Fixed and extended description for target classes in the test&score's documentation.

##### Includes
- [ ] Code changes
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: doc/visual-programming/source/widgets/evaluation/testandscore.rst [hunk 1]
```diff
@@ -52,10 +52,14 @@ than one widget to test multiple learners with the same procedures.
       we select *Separate Test Data* signal in the communication channel
       and select Test on test data.
 
-2. Only *Test on test data* requires a target class, e.g. having the
-   disease or being of subvariety *Iris setosa*. When *Target class* is
-   (None), the methods return the average value. Target class can be
-   selected at the bottom of the widget. 
+2. For the classification *Target class* can be selected at the bottom of the widget.
```
Comment (lanzagar) [inline doc/visual-programming/source/widgets/evaluation/testandscore.rst]: Not sure about the best wording, but maybe something like:
For classification, *Target class* can...

Comment (PrimozGodec) [reply doc/visual-programming/source/widgets/evaluation/testandscore.rst]: Fixed. If anybody has any other ideas on how to improve the text welcome is welcome to make a suggestion.

File: doc/visual-programming/source/widgets/evaluation/testandscore.rst [hunk 2]
```diff
@@ -52,10 +52,14 @@ than one widget to test multiple learners with the same procedures.
       we select *Separate Test Data* signal in the communication channel
       and select Test on test data.
 
-2. Only *Test on test data* requires a target class, e.g. having the
-   disease or being of subvariety *Iris setosa*. When *Target class* is
-   (None), the methods return the average value. Target class can be
-   selected at the bottom of the widget. 
+2. For classification, can *Target class* can be selected at the bottom of the widget.
```
Comment (lanzagar) [inline doc/visual-programming/source/widgets/evaluation/testandscore.rst]: Erm, sorry but now there is one `can` too many.

Comment (PrimozGodec) [reply doc/visual-programming/source/widgets/evaluation/testandscore.rst]: fixed

**Pull request merged**